google-cloud-pubsub/lib/google/cloud/pubsub/topic.rb

```diff
@@ -51,6 +52,33 @@ def initialize
           @grpc = Google::Pubsub::V1::Topic.new
           @name = nil
           @exists = nil
```

Comment:
Seems inconsistent that the subscriber is its own file but the publisher is under topic. I would prefer that Publisher is separate from Topic, for consistency within Ruby and with other client libraries.